samples/subsys/ipc/ipc_service/static_vrings/sample.yaml

```
@@ -3,6 +3,9 @@ sample:
 tests:
   sample.ipc.static_vrings:
     filter: dt_compat_enabled("zephyr,ipc-openamp-static-vrings")
+    platform_exclude:
+      - nrf5340dk/nrf5340/cpuapp_ns
```

Please rename the platform:
```suggestion
      - nrf5340dk/nrf5340/cpuapp/ns
```

---

Done